_rules/html-page-title-descriptive-c4a8a4.md

```diff
@@ -127,7 +127,7 @@ Even though the descriptive `<title>` element is not placed within the `<head>`
 
 #### Failed Example 2
 
-Even though a correct `<title>` element is put in the `<head>` of the document, this rule only looks at the first `<title>` element.
+This `<title>` element, the first of two, does not describe the content of the document. Most browsers, and this rule, only look at the first `<title>` element.
```

Comment:
Same, this is also likely better suiting for the "page has title" rule.

Reply:
Again, possibly a separate issue to this PR.

Reply:
@Jym77 did you create an issue for your concerns with this rule, or should I?

Reply:
I didn't.